CareKit/SymptomTracker/OCKSymptomTrackerViewController.m

```diff
@@ -84,8 +84,7 @@ - (void)viewDidLoad {
                                                                               style:UIBarButtonItemStylePlain
                                                                              target:self
                                                                              action:@selector(showToday:)];
-    self.navigationItem.rightBarButtonItem.tintColor = self.progressRingTintColor;
-    
```

Comment:
Yes, I confirm you have understood in the right manner, but at this very moment it seems that rightBarButtonItem stays always with the default tintColor (blue) with the current implementation in master repository, I was forced to tweak the code like PR above to have it working.